### ISSUE TITLE
train_accuracy

### ISSUE THREAD

**Issue by nightfuryyy** [xuexingyu24/License_Plate_Detection_Pytorch#4]
i tried to train my model on my plate data.

"loss: 0.0000, train_accuracy: 0.1875, time: 0.28 s/iter, learning rate: 0.0001"

loss so small, but train_accuracy always <= 0.4 . I trained 5hour and can't get better result. How can i fix that ?

**Comment by xuexingyu24** [comment]
@nightfuryyy   how did you initialize the weights during training ?  I will recommend you to use pretrained weights from [sirius's repo](https://github.com/sirius-ai/LPRNet_Pytorch) as an initializer.

**Comment by nightfuryyy** [comment]
my classnum is 31. So i can't use his pretrained weights. Btw i generated gray images for training.

**Comment by wait1988** [comment]
只是用了ccpd来训练吗，从模型测试效果来看，感觉数据很不均衡 @xuexingyu24

**Comment by kunyao2015** [comment]
same problem , initialize the weights random

**Comment by persuelx** [comment]
@nightfuryyy @kunyao2015 your problem has solved?I had the same problem when I trained my custom data。
